<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPageWidget</class>
 <widget class="QWidget" name="SettingsPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(247, 247, 247);</string>
  </property>
  <widget class="QLabel" name="label_colorScheme">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>外观</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_bright">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>28</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/Share/images/ColorScheme_BrightMode_BrightBtn1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_dark">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>28</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/Share/images/ColorScheme_BrightMode_DarkBtn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_bright">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>140</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_dark">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>140</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_auto">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>189</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>自动化</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_auto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>197</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_auto_hint">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>197</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>勾选此项时,Cell会自动根据系统时间设定相应的颜色模式。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
